libraries/RC_Channel/RC_Channel.h

```diff
@@ -395,6 +395,9 @@ class RC_Channels {
     // this function is implemented in the child class in the vehicle
     // code
     virtual RC_Channel *channel(uint8_t chan) = 0;
+    RC_Channel *lua_rc_channel(const uint8_t chan) {
```

Comment:
maybe add a comment above this method to make it clear what it's for.

Reply:
done